tests/e2e.sh

```diff
@@ -23,12 +23,12 @@ case $(uname -m) in
 	*)		ARCH="$(uname -m)";;
 esac
 
-KUBERNETES_VERSION=v1.19.1
+KUBERNETES_VERSION=v1.19.2
```

Comment:
unfortunately, kind's docker images don't have the same release cadence like that of kubernetes.

Reply:
Thanks, yes figured it after CI run and dropped the change now.